examples/simple-lambda-worker/README.md

```diff
@@ -0,0 +1,52 @@
+# LambdaWorker Example: SimpleLambdaWorker
+
+This is an example of using the LambdaWorker in a simple worker.
+
+The LambdaWorker:
+
+- is subscribed to an SQS queue whose trigger is enabled by default
+- sends failed jobs to another SQS queue - a dead letter queue
+- is subscribed to the dead letter queue but this trigger is disabled by default
+- allows for the replay of failed jobs with the talis-cli tools awsdlq tool
+- triggers an alarm for failed jobs
+- triggers an alarm when the queue of jobs exceeds a configuarble maximum threshold
+- triggers an alarm when a message waiting on the queue is older than a configurable maximum
+
+Optionionally, the LambdaWorker:
+
+- can have its SQS queue subscribed to an SNS topic.
+  - For example all Depot workers subscribe to a "File" topic in a Pub/Sub architecture.
+- can have a filterPolicy specified so that it only receives messages from the SNS topic which it is interested in.
+
+## Construct Diagram
+
+![LambdaWorker Construct diagram](LambdaWorker.drawio.png)
+
+## Note:
+
+The `talis-cli-tools` `awsdlq` tool can be used to replay messages.
+See the tech team talk on the `awsdlq` [here](https://docs.google.com/presentation/d/1ND9LMPBeMFqqoxCfsh3Z57Gl0N8JnA9XjZGmqYgTo_A).
+
+## This Example
+
+To build and deploy this example:
+
+- `export AWS_PREFIX=development-XX-` where XX are your initials
```

Comment:
The [(new) docs](https://github.com/talis/infra/wiki/Infrastructure-Best-Practices) suggest `dev-XX`, which is better - there's a character limit on things that the prefix will touch, and that's a lot fewer chars!